### PR TITLE
Added some very basic generation instructions for Windows.

### PR DESCRIPTION
**Pull request by Falven**
Had some confusion. Was trying to generate with PSCore and even CMD Prompt before I looked through the CI script/Makefile and made the connection to use Git Bash for Windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,15 @@ For more information on API usage, please refer to [src/native/api.h](https://gi
 * Python: [pallas/pyllhttp][8]
 * Ruby: [metabahn/llhttp][9]
 
+## Generating on Windows
+
+* choco install git
+* choco install node
+* choco install llvm
```
Comment (indutny) [inline README.md:106]: Do you still need `llvm` for building `llhttp`? I think this shouldn't be a requirement anymore.

**Review by indutny — APPROVED** [review]
LGTM with a question. Great work!

File: README.md (hunk 1 above)
Comment (Falven) [reply README.md:106]: lol. I'm actually having issues linking the object file under Windows. I created an issue [here](https://github.com/nodejs/llhttp/issues/109).
If I can find a way to get this issue resolved I update these instructions with more detail.

**Comment by Falven** [comment]
lol. I'm actually having issues linking the object file under Windows. I created an issue [here](https://github.com/nodejs/llhttp/issues/109).
If I can find a way to get this issue resolved I update these instructions with more detail.

**Comment by Falven** [comment]
Figured out the issue and added some better instructions. LMK what you think.

File: README.md (hunk 1 above)
Comment (Falven) [reply README.md]: It likely will work with other compilers. I've just tried with clang and it worked.

**Review by indutny — APPROVED** [review]
LGTM

**Pull request merged**